src/components/ShowContextMenuContext.js

```diff
@@ -20,7 +20,7 @@ ShowContextMenuContext.displayName = 'ShowContextMenuContext';
  * @param {Object} action - ReportAction for ContextMenu
  * @param {Function} checkIfContextMenuActive Callback to update context menu active state
  */
-function showContextMenuForReport(event, anchor, reportID, action, checkIfContextMenuActive) {
+function showContextMenuForReport(event, anchor, reportID, action, checkIfContextMenuActive, isArchivedRoom = false) {
```

Comment:
Please add [JSDOC](https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#jsdocs) for `isArchivedRoom` parameter

Reply:
Done